.npmignore

```diff
@@ -0,0 +1,3 @@
+coverage/
```

Comment:
Not 100% sure but I don't think this needs to be in .npmignore. This should certainly be in .gitignore and I believe .npmignore ignores anything ignored by git. You can quickly check it by running npm pack locally

Reply:
It's in the .gitignore but was still being included when I ran `npm pack`. I can remove it though

Reply:
fine to leave it here then